rfc-126-custom-configuration-for-dependabot.md

```diff
@@ -187,7 +189,7 @@ update_configs:
 
 Most of our applications use the same internal and framework libraries, so we'll want a single global configuration across all our apps. At the same time, there might be some libraries used by particular apps which fit within one of the three categories.
 
-Maintaining this file across each of our apps has the potential for it to become out of date with our global configuration. To solve this problem, this RFC proposes that we add functionality to [govuk-saas-config] which ensures the configuration file in each repo is up to date with a single global configuration file. Keeping up to date means ensuring that the individual repo configuration file contains at least the global configuration options plus any extra custom configuration for the app itself.
+Maintaining this file across each of our apps has the potential for it to become out of date with our global configuration. To solve this problem, this RFC proposes that we add functionality to [govuk-saas-config] which ensures the configuration file in each repo is up to date with a single global configuration file. Keeping up to date means ensuring that the individual repo configuration file contains at least the global configuration options plus any extra custom configuration for the app itself. This should allow some apps to include framework libraries that are used by specific apps (for example `sinatra`) but not used wider across all our apps.
```

Comment:
```suggestion
Maintaining this file across each of our apps has the potential for it to become out of date with our global configuration. To solve this problem, this RFC proposes that we add functionality to [govuk-saas-config] which ensures the configuration file in each repo is up to date with a single global configuration file. Keeping up to date means ensuring that the individual repo configuration file contains at least the global configuration options plus any extra framework libraries that are used by specific apps (for example `sinatra`) but not used wider across all our apps.
```

Reply:
Pretty minor: the ambiguity before is whether the app could also tweak the top-level configuration, which shouldn't be necessary. I can't see why an app would need to tweak the config apart from for framework dependencies, so it's helpful if we can be more specific and say that.

Reply:
I've added your suggestion in 19c6952 which I think makes this clearer, thanks!